<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="factorialButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate factorial</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rangeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="primeButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate primes</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
